site/generate-jenkins-io-template.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+WWW_ROOT_DIR="${WWW_ROOT_DIR:-${1:-./www2}}"
+SELF_URL="${SELF_URL:-${2:-https://updates.jenkins.io}}"
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+wget --quiet --convert-links --output-document "$WWW_ROOT_DIR/index.html" --convert-links https://www.jenkins.io/templates/updates/index.html
+sed -i.bak -e "s|<jio-navbar.*/jio-navbar>|<jio-navbar class='fixed-top' id='ji-toolbar' property='${SELF_URL}'></jio-navbar>|g" "$WWW_ROOT_DIR/index.html"
```

Comment:
does mac allow `-i''`? There's no need for a backup since the file is pulled down seconds earlier
would it be better to switch to `perl -pi -e`, i know that works almost everwhere.

Honestly it probably could just be pipes, output to stdout, then pipe to sed (optionally two seds) then write to file

Reply:
`-i ''` (note the space, `-i''` is just the same as `-i`) would work on Mac OS, but not Linux.